votor/src/event_handler.rs

```diff
@@ -197,6 +217,27 @@ impl EventHandler {
         Ok(())
     }
 
+    fn maybe_send_genesis_vote(
+        migration_status: &MigrationStatus,
+        vctx: &mut VotingContext,
+    ) -> Result<bool, EventLoopError> {
+        let Some((slot, block_id)) = migration_status.get_votable_genesis_block() else {
+            return Ok(false);
+        };
+
+        let vote = Vote::new_genesis_vote(slot, block_id);
+        let Some(bls_op) = generate_vote_message(
+            vote,
+            // We set `is_refresh` because we don't want this vote in the `VoteHistory`
+            true, vctx,
```

Comment:
nit: we probably should rename `is_refresh` to `do_not_update_vote_history` or something?